server/middleware/rest.js

```diff
@@ -55,14 +56,8 @@ function rest() {
       }
 
       if (app.isAuthEnabled) {
-        // NOTE(bajtos) It would be better to search app.models for a model
-        // of type AccessToken instead of searching all loopback models.
-        // Unfortunately that's not supported now.
-        // Related discussions:
-        // https://github.com/strongloop/loopback/pull/167
-        // https://github.com/strongloop/loopback/commit/f07446a
-        var AccessToken = loopback.getModelByType(loopback.AccessToken);
-        handlers.push(loopback.token({ model: AccessToken }));
+        var AccessToken = registry.getModelByType(registry.getModel('AccessToken'));
```

Comment:
As I mentioned in past, this is IMO a common use case. Please improve `registry.getModelByType` to accept both model constructor and model name.